torchrl/data/tensordict/tensordict.py

```diff
@@ -1,4 +1,4 @@
-#pad_size, value Copyright (c) Meta Platforms, Inc. and affiliates.
+# pad_size, value Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
`pad_size, value` string should be deleted maybe ?

Reply:
😅 yes they are now